notebooks/tutorial.ipynb

```diff
@@ -0,0 +1,433 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "session.SessionDirectory.insert1(\n",
+    "    dict(**session_key,\n",
+    "        session_dir=\"subject1/session1\")\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scan.Scan.insert1(\n",
+    "    dict(\n",
+    "        **session_key,\n",
+    "        scan_id=0,\n",
+    "        scanner=\"ScanImage\",\n",
+    "        acq_software=\"ScanImage\",\n",
+    "        scan_notes=\"\",\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "populate_settings = {\n",
+    "                    \"display_progress\": True,\n",
+    "                    \"reserve_jobs\": False,\n",
+    "                    \"suppress_errors\": False,\n",
+    "                    }"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scan.ScanInfo.populate(**populate_settings)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import suite2p\n",
+    "\n",
+    "params_suite2p = suite2p.default_ops()\n",
+    "params_suite2p['nonrigid']=False\n",
+    "\n",
+    "imaging.ProcessingParamSet.insert_new_params(\n",
+    "    processing_method=\"suite2p\",\n",
+    "    paramset_idx=0,\n",
+    "    params=params_suite2p,\n",
+    "    paramset_desc=\"Calcium imaging analysis with Suite2p using default Suite2p parameters\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "imaging.ProcessingTask.insert1(\n",
+    "    dict(\n",
+    "        **session_key,\n",
+    "        scan_id=0,\n",
+    "        paramset_idx=0,\n",
+    "        processing_output_dir=\"subject1/session1/suite2p\",\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "imaging.Processing.populate(**populate_settings)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "imaging.Curation.insert1(\n",
+    "    dict(\n",
+    "        **session_key,\n",
+    "        scan_id=0,\n",
+    "        paramset_idx=0,\n",
+    "        curation_id=0,\n",
+    "        curation_time=\"2021-04-30 12:22:15.032\",\n",
+    "        curation_output_dir=\"subject1/session1/suite2p\",\n",
+    "        manual_curation=False,\n",
+    "        curation_note=\"\",\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "imaging.MotionCorrection.populate(**populate_settings)"
```

Comment:
Got an error on this step when I ran it:

```
KeyError                                  Traceback (most recent call last)
/tmp/ipykernel_3552/1818698745.py in <module>
----> 1 imaging.MotionCorrection.populate(**populate_settings)

~/.local/lib/python3.7/site-packages/datajoint/autopopulate.py in populate(self, suppress_errors, return_exception_objects, reserve_jobs, order, limit, max_calls, display_progress, processes, make_kwargs, *restrictions)
    239                 tqdm(keys, desc=self.__class__.__name__) if display_progress else keys
    240             ):
--> 241                 error = self._populate1(key, jobs, **populate_kwargs)
    242                 if error is not None:
    243                     error_list.append(error)

~/.local/lib/python3.7/site-packages/datajoint/autopopulate.py in _populate1(self, key, jobs, suppress_errors, return_exception_objects, make_kwargs)
    290                 self.__class__._allow_insert = True
    291                 try:
--> 292                     make(dict(key), **(make_kwargs or {}))
    293                 except (KeyboardInterrupt, SystemExit, Exception) as error:
    294                     try:

~/.local/lib/python3.7/site-packages/element_calcium_imaging/imaging.py in make(self, key)
    799                             "block_width": s2p.ops["block_size"][1],
    800                             "block_depth": 1,
--> 801                             "block_count_y": s2p.ops["nblocks"][0],
    802                             "block_count_x": s2p.ops["nblocks"][1],
    803                             "block_count_z": len(suite2p_dataset.planes),

KeyError: 'nblocks'
```

Ran it again with your original setting for `imaging_root_data_dir` and got:

```
FileNotFoundError                         Traceback (most recent call last)
/tmp/ipykernel_19561/1299329365.py in <module>
----> 1 imaging.Processing.populate(**populate_settings)

~/.local/lib/python3.7/site-packages/datajoint/autopopulate.py in populate(self, suppress_errors, return_exception_objects, reserve_jobs, order, limit, max_calls, display_progress, processes, make_kwargs, *restrictions)
    239                 tqdm(keys, desc=self.__class__.__name__) if display_progress else keys
    240             ):
--> 241                 error = self._populate1(key, jobs, **populate_kwargs)
    242                 if error is not None:
    243                     error_list.append(error)

~/.local/lib/python3.7/site-packages/datajoint/autopopulate.py in _populate1(self, key, jobs, suppress_errors, return_exception_objects, make_kwargs)
    290                 self.__class__._allow_insert = True
    291                 try:
--> 292                     make(dict(key), **(make_kwargs or {}))
    293                 except (KeyboardInterrupt, SystemExit, Exception) as error:
    294                     try:

~/.local/lib/python3.7/site-packages/element_calcium_imaging/imaging.py in make(self, key)
    386 
    387         if task_mode == "load":
--> 388             method, imaging_dataset = get_loader_result(key, ProcessingTask)
    389             if method == "suite2p":
    390                 if (scan.ScanInfo & key).fetch1("nrois") > 0:
...
---> 52                 "Suite2p output result files not found at {}".format(suite2p_dir)
     53             )
     54 

FileNotFoundError: Suite2p output result files not found at /workspaces/workflow-calcium-imaging/example_data/subject1/session1/suite2p
```

Reply:
This was an issue with the data on S3.  It is now resolved.